client/my-sites/plugins/plugins-discovery-page/index.jsx

```diff
@@ -89,6 +109,7 @@ const PluginsDiscoveryPage = ( props ) => {
 	} );
 
 	const isLoggedIn = useSelector( isUserLoggedIn );
+	const isWPBeginnerSpecial = getQueryArgs()?.ref === 'wpbeginner-special-lp';
```

Comment:
TODO: Replace this with a check for `site_partner_bundle` blog option once  #95551 is merged.